ui/guest/package.json

```diff
@@ -20,6 +20,7 @@
     "react-dom": "^16.12.0",
     "react-scripts": "3.3.0-next.62",
     "rxjs": "^6.5.3",
+    "uppy": "^1.7.0",
```

Comment:
As mentioned, let's do the update Studio-side. So you may `yarn remove` this change (rollback).